project/CONTRIBUTING.md

```diff
@@ -87,9 +87,16 @@ Some rules have to be respected about the test:
   * `@codeCoverageIgnore`
   * `@codeCoverageIgnoreStart`
   * `@codeCoverageIgnoreEnd`
+  * `@expectedException`
+  * `@expectedExceptionMessages`
 * All test methods should be prefixed by `test`. Example: `public function testItReturnsNull()`.
+* All test method names must be in camel case format
 * As opposed, the `@test` annotation is prohibited.
 * Most of the time, the test class should have the same name as the targeted class, suffixed by `Test`.
+* `@expectedException*` annotations are prohibited in favor of:
+  * the `setExpectedException` method if phpunit 4 is still in use;
+  * the `expectException*` methods otherwise.
```

Comment:
No, it is methods (notice the star, I mean `expectException()` AND `expectExceptionMessage()`, etc…